frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/providers/ovirt/ovirt-prefill-vm.ts

```diff
@@ -90,15 +90,25 @@ export const getDisks = (vm: OvirtVM, storageClassConfigMap: ConfigMapKind): VMW
 
 export const getNics = (vm: OvirtVM): VMWizardNetwork[] => {
   const getUniqueName = createUniqueNameResolver(vm.nics);
-  const nics = (vm.nics || []).filter((n) => n);
+  const nics = (vm.nics || []).filter(
+    (nic) => nic && nic.name && nic.vnicid != null && nic.vnicid !== '',
```

Comment:
can we use the ? notation here ?
`nic?. ...1`

Reply:
fixed